src/domain-services/software-info/software-info-service.ts

```diff
@@ -0,0 +1,14 @@
+import { Service } from "typedi";
+import SoftwareInfo from "./graphql/types";
+import  {version} from '../../../package.json';
+
+@Service()
+export class SoftwareInfoService {
+    async getSoftwareInfo(): Promise<SoftwareInfo[]> {
+        return [{
+            title: 'HPC GraphQL API',
+            status: 'Stable',
+            version,
+        }]
+    }
```

Comment:
This method is not async. Once you apply the proposed changes from below, they will propagate to the caller, which also doesn't need to be async.
```suggestion
    getSoftwareInfo(): SoftwareInfo[] {
        return [{
            title: 'HPC GraphQL API',
            status: 'Stable',
            version,
        }]
    }
```

Reply:
I've apply this changes and needed one in the caller